docs/guides/events-list.rst

```diff
@@ -65,6 +65,12 @@ System events
 **cache:flush, system**
     Reset internal and external caches, by default including system_cache, simplecache, and memcache. One might use it to reset others such as APC, OPCache, or WinCache.
 
+**send:before, http_response**
```

Comment:
Could this go in a separate PR? Would be nice to assemble use cases and cancellation behavior.